src/styled/globalStyle.tsx

```diff
@@ -0,0 +1,17 @@
+import  { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
you should check out if it is possible to compose global styles.
It could then maybe look like this:
const GlobalStyle = styled(GlobalSizes, GlobalFonts)

Reply:
"createGlobalStyle" it's not same as "styled".
styled() extend basic styles of component or it can change some properties. it can works a little like className selector
createGlobalStyle can inject style to all selectors, to change its base,.  when we use somewar  'nav', 'ul' or 'button' it always return as this changed version.

thats is how i understand how to use styledComponents xd

Reply:
ya, ya.
But I wonder if there is some way to make a composite of different styles and them apply the sum of them.
something like maybe createGlobalStyle(GlobalSizes, GlobalFonts)